vars/DevelopPipeline.groovy

```diff
@@ -129,10 +137,10 @@ def call(Map pipeline_args = [:]) {
         post {
             always {
                 dir("${env.WORKSPACE}/repo/${pipeline_args.name}") {
-                    junit 'jenkinsReport/*.xml'
+                    junit allowEmptyResults: true, testResults: 'jenkinsReport/*.xml'
 
                     publishHTML (target: [
-                    allowMissing: false,
+                    allowMissing: true,
```

Comment:
This allows the report to be missing without failing the build by default, needed for the code coverage skip since there's no report if coverage is not run.